core/unit_test/TestTeamVector.hpp

```diff
@@ -624,7 +731,7 @@ struct functor_vec_single {
         Kokkos::ThreadVectorRange(team, nStart, nEnd),
         [&](int /*i*/, Scalar &val) { val += value; }, value2);
 
-    if (value2 != (value * (nEnd - nStart))) {
+    if (value2 != (value * Scalar(nEnd - nStart))) {
```

Comment:
How does that matter?

Reply:
some ambiguity I needed to get rid off ...